### PR TITLE
[wasm][sdk] Add dotnet new custom template.

### PR DESCRIPTION
**Pull request by kjpou1**
```
Templates                                         Short Name         Language          Tags
----------------------------------------------------------------------------------------------------------------------------
Mono WebAssembly Starter                          wasm               [C#]              Mono/WebAssembly
```

Example use:

```
dotnet new wasm -n Test.WebAssembly -o Test.WebAssembly
```

### CODE REVIEW

File: sdks/wasm/sdk/Templates/DotNet/.template.config/template.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+    "author": "Mono",
+    "classifications": [ "Mono/WebAssembly" ],
+    "name": "Mono WebAssembly Starter",
+    "identity": "WasmTemplate.1",
```
Comment (kg) [inline sdks/wasm/sdk/Templates/DotNet/.template.config/template.json]: is the 1 significant in any way?

Comment (kjpou1) [reply sdks/wasm/sdk/Templates/DotNet/.template.config/template.json]: No.  I was just following the naming convention suggestion doc.

**Comment by kjpou1** [comment]
@monojenkins build failed

File: sdks/wasm/sdk/Templates/DotNet/.template.config/template.json (hunk 1 above)
Comment (kjpou1) [reply sdks/wasm/sdk/Templates/DotNet/.template.config/template.json]: https://github.com/dotnet/templating/wiki/Naming-and-default-value-forms

**Comment by kjpou1** [comment]
@monojenkins build failed

**Comment by kjpou1** [comment]
@monojenkins build Linux x64

**Pull request merged**